src/priority_q.py

```diff
@@ -0,0 +1,32 @@
+from bin_heap import BinHeap
+
+
+class PQueue(object):
+    """Priority Queue object."""
+
+    def __init__(self):
+        """Initialize priority queue object."""
+        self._bh = BinHeap()
+        self._heap = self._bh._heap
+        self._count = 0
+
+    def insert(self, priority, data):
```

Comment:
Good.  I like that you let the user enter the data and the priority themselves. One consideration might be to let `priority` be a optional keyword argument, with some low default number as priority.